config/config.go

```diff
@@ -64,6 +64,8 @@ type Configuration struct {
 	AccountRequired bool `mapstructure:"account_required"`
 	// Local private file containing SSL certificates
 	PemCertsFile string `mapstructure:"certificates_file"`
+
+	CustomHeaders CustomHeaders `mapstructure:"custom_headers"`
```

Comment:
Consider adding a comment here to describe the use of `CustomHeaders`

Reply:
Added. Please review it again